test/dummy/app/components/card/card_component.rb

```diff
@@ -0,0 +1,19 @@
+class CardComponent < MountainView::Presenter
+  attributes :title, :description, :link, :image_url
+  attribute :data, default: []
+  attribute :foobar
+
+  def title
+    ['Foobar', properties[:title]].join(' - ')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.